osu.Game/Localisation/ChatStrings.cs

```diff
@@ -24,6 +24,11 @@ public static class ChatStrings
         /// </summary>
         public static LocalisableString MentionUser => new TranslatableString(getKey(@"mention_user"), @"Mention");
 
+        /// <summary>
+        /// "Report"
+        /// </summary>
+        public static LocalisableString ReportUser => new TranslatableString(getKey(@"report_user"), @"Report");
```

Comment:
There's a `ReportStrings.UserButton` existing on web, but not sure since there's no specific string for chat (i.e. that string is for general "user" reporting).